2019=>2020 Changes

tswfi opened the pull request:
untested wip

tswfi committed:
Add gemfiles for easier local testing

tswfi committed:
gemfiles please, no need for jekyll-cache

tswfi committed:
gh-pages has jekyll-default-layout enabled by default
enable explicitly for easier local testing

tswfi committed:
And finally the readme for others :)

tswfi committed:
started 2019=>2020 changes

tswfi committed:
started 2019=>2020 changes

tswfi committed:
few 2019 images

tswfi committed:
few 2019 images

tswfi committed:
Merge pull request #3 from tswfi/AddReadme

Readme and some changes to get local jekyll up and running easier

tswfi committed:
Merge branch 'year2020' of github.com:tswfi/robotit into year2020

vranki reviewed (approved):
LGTM, vaikka en osaakkaan

tswfi committed:
comment out 2019 fb event, add config for 2020 fb event for someone to fill later

tswfi committed:
remove note about possible track changes in 2019

tswfi committed:
Add a quick menu

otherwise coming back from the rules page is not possible and there is no links at all to the english or swedish frontpage...

tswfi committed:
comment out todo section in 2019

tswfi committed:
fix isodates to correct year

tswfi commented:
@jssmk teknisesti ok, mutta jos haluat sisältöön vaikuttaa niin kerro (esim kellonaika on nyt vaan copy pastettu edellisestä vuodesta)

jssmk commented:
Testattu, hyvältä näyttää
